airsonic-main/src/main/java/org/airsonic/player/dao/AbstractDao.java

```diff
@@ -69,11 +73,19 @@ protected String prefix(String columns, String prefix) {
         l.replaceAll(s -> prefix + "." + s);
         return String.join(", ", l);
     }
+    
+    protected static Object[] convertArgs(Object[] args) {
+        return args == null ? null : Stream.of(args).map(x -> (Object) ((x instanceof Instant) ? Timestamp.from((Instant) x) : x)).collect(Collectors.toList()).toArray();
+    }
+    
+    protected static Map<String, Object> convertArgs(Map<String, Object> args) {
```

Comment:
I'd name the method better than `convertArgs` for example `convertToDatabaseTypes`?

Reply:
👍 I'll rename